src/harness/testcases/WINNF_FT_S_SSS_testcase.py

```diff
@@ -18,10 +18,67 @@
 
 SAS_CERT = os.path.join('certs', 'sas.cert')
 SAS_KEY = os.path.join('certs', 'sas.key')
+SAS_URL = 'https://fake.sas.url.not.used.org'
```

Comment:
The SAS_URL should also include the version number of the SAS.

Reply:
Done.